### DISCUSSION TITLE
A postman collection

### DISCUSSION THREAD

**Discussion by dcw-byteforged** [umputun/remark42#1665] [Ideas]
Hey guys, I was browsing through the documentation and I noticed that content for API section was sparse. 
I was wondering if we could perhaps maybe get a postman collection for the functionality of this product.

If there is some better documentation for the API section, I would be thrilled if you can guide me to it.

Personally I like what this product has to offer and I do want to see if we can adopt using this product instead of the alternatives.

**Comment by umputun** [comment]
if you think what API content is sparse, feel free to submit a PR with improvements. Regarding postman thing - it already has [remark.rest](https://github.com/umputun/remark42/blob/master/backend/remark.rest) file with a lot of API samples. Not sure if postman can import it, but this thing works fine with rest clients for both VSCode and IDEA
